db/dcrpg/upgrades.go

```diff
@@ -673,7 +675,9 @@ func (pgb *ChainDB) handleAgendasVotingMilestonesUpgrade() (int64, error) {
 			if val > 0 {
 				query := fmt.Sprintf("UPDATE agendas SET %v=true WHERE block_height=$1 AND agenda_id=$2", name)
 				_, err := pgb.db.Exec(query, val, id)
-				errorLog(name, id, err)
+				if errorLog(name, id, err) {
+					return rowsUpdated, err
```

Comment:
A non-nil error value returning from from `handleAgendasVotingMilestonesUpgrade` was impossible.  If an update statement fails, the error was supposed to stop this and trickle up.